rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
I think the motivation for having `app` trimmed from the path for the spec files is when you have subdirectories of `app`, like `models`. So when you have `myapp/app/models/foo.rb` the spec path is `myapp/spec/models/foo.rb`. Not so much for loose files directly in `spec`.